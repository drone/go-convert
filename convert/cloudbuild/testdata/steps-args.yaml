# https://cloud.google.com/build/docs/build-config-file-schema#args

steps:
- name: 'gcr.io/cloud-builders/mvn'
  args: ['install']
- name: 'gcr.io/cloud-builders/docker'
  args: ['build', '-t', 'gcr.io/my-project-id/myimage', '.']