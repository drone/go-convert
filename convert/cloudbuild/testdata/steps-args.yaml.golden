kind: pipeline
spec:
  options:
    envs:
      BRANCH_NAME: <+trigger.branch>
      BUILD_ID: <+pipeline.sequenceId>
      COMMIT_SHA: <+trigger.commitSha>
      PROJECT_ID: <+project.name>
      PROJECT_NUMBER: <+pipeline.sequenceId>
      REPO_NAME: <+trigger.payload.repository.name>
      REVISION_ID: <+trigger.commitSha>
      SHORT_SHA: <+codebase.shortCommitSha>
      TAG_NAME: <+trigger.commitSha>
  stages:
  - desc: converted from google cloud build
    name: pipeline
    spec:
      platform:
        arch: amd64
        os: linux
      runtime:
        spec: {}
        type: cloud
      steps:
      - name: mvn
        spec:
          args:
          - install
          image: gcr.io/cloud-builders/mvn
          mount:
          - name: dockersock
            path: /var/run/docker.sock
        type: script
      - name: docker
        spec:
          args:
          - build
          - -t
          - gcr.io/my-project-id/myimage
          - .
          image: gcr.io/cloud-builders/docker
          mount:
          - name: dockersock
            path: /var/run/docker.sock
        type: script
      volumes:
      - name: dockersock
        spec:
          path: /var/run/docker.sock
        type: host
    type: ci
version: 1
